docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
 
-.. math::
+    active_model = lk.ActiveCalibrationModel(driving_data.data, volts.data, driving_data.sample_rate, bead_diameter, driving_frequency_guess=38, distance_to_surface=distance_to_surface)
 
-    P^\mathrm{thermal}(f) = \frac{D}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)}
+To check the determined frequency, we can look at the determined driving frequency::
 
-    P^\mathrm{active}(f) = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_\mathrm{drive}^2}\right)} \delta(f - f_\mathrm{drive})
+    >>> active_model.driving_frequency
+    38.15193077664577
 
-    P^\mathrm{total}(f) = P^\mathrm{thermal}(f) + P^\mathrm{active}(f)
+Let's have a look to see if this peak indeed appears in our power spectrum.
+To see it clearly, we reduce the blocking amount and show the spectrum all the way up to a frequency of 10 Hz::
 
-Since we know the driving amplitude, :math:`A`, we know how the bead reacts to the driving motion and we can observe this response in the power spectral density (PSD), we can use this relation to determine the positional calibration.
+    show_peak = lk.calculate_power_spectrum(volts.data, sample_rate=78125, num_points_per_block=5, fit_range=(10, 23000))
+    show_peak.plot()
 
-If we use the basic Lorentzian model, then the theoretical power (integral over the delta spike) corresponding to the driving input is given by :cite:`tolic2006calibration`:
+.. image:: figures/force_calibration/calibration_peak.png
 
-.. math::
+The driving peak is clearly visible in the spectrum.
+Let's calculate the power spectrum we'll use for fitting.
+It is important to _not_ include the driving peak when doing this (the default will only go up to 100 Hz)::
 
-    W_\mathrm{physical} = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_\mathrm{drive}^2}\right)} \quad \mathrm{[m^2]}
+    power_spectrum = lk.calculate_power_spectrum(volts.data, sample_rate=78125)
 
-Subtracting the thermal part of the spectrum, we can determine the same quantity experimentally.
+We can now use this to fit our data::
 
-.. math::
+    calibration = lk.fit_power_spectrum(power_spectrum, active_model)
+    calibration
 
-    W_\mathrm{measured} = \left(P_\mathrm{measured}^\mathrm{total}(f_\mathrm{drive}) - P_\mathrm{measured}^\mathrm{thermal}(f_\mathrm{drive})\right) \Delta f \quad \mathrm{[V^2]}
+.. image:: figures/force_calibration/calibration_item_active.png
 
-:math:`\Delta f` refers to the width of one spectral bin.
-Here the thermal contribution that needs to be subtracted is obtained from fitting the thermal part of the spectrum using the passive calibration procedure from before.
-The desired positional calibration is then:
+Analogous to the passive calibration procedure, we can specify `hydrodynamically_correct=True` if we wish to use the hydrodynamically correct theory here.
+Especially for bigger beads this is highly recommended (more on this later).
 
-.. math::
+More convenient calibration
+---------------------------
```

Comment:
I don't know there's a good way to do this, but I feel this section could be almost at the top. It'd be nice right away to show users that they don't *need* to go through the hassle of setting up the models manually. 

Perhaps move the first half of this section toward the top as a TLDR-ish part and then take the 2nd half here comparing different models as a "Comparison" section here still?

Reply:
I felt the same way when writing this but held off on moving it because I didn't want to fracture the introduction and use of the convenience function. I do agree though, and will attempt to move it up.